www/authmemcookie.php

```diff
@@ -91,12 +91,25 @@
 	/* Store the authentication data in the memcache server. */
 
 	$data = '';
+	$amcConfig = SimpleSAML_Configuration::getConfig('authmemcookie.php');
+	$format = strtolower($amcConfig->getBoolean('format', 'implode'));
+	assert('in_array($format, array("implode", "multi", "json"))');
```

Comment:
I don't think we should use an `assert()` here. Asserts are intended for debugging, and we are already abusing them all over the place. If the assertion fails, it would be due to a configuration mistake, not a development mistake, so we should throw an exception.